vertexai/main.ts

```diff
@@ -0,0 +1,29 @@
+import { initializeApp } from 'firebase/app';
+import { firebaseConfig, RECAPTCHA_ENTERPRISE_SITE_KEY } from './config';
+import {
+  initializeAppCheck,
+  ReCaptchaEnterpriseProvider,
+} from "firebase/app-check";
+import { getVertexAI, getGenerativeModel } from "firebase/vertexai-preview";
+
+async function main() {
+  const app = initializeApp(firebaseConfig);
+
+  // initialize app check
+  initializeAppCheck(app, {
```

Comment:
Non blockign nit: I feel like AppCheck is a barrier/burden for new users. Isn't this optional for the quickstart? Are we including it here becuase our recommendation is that products use AppCheck for actual releases? I feel like maybe we should make a note of that somehwere in a comment here, since this is a quickstart. WDYT?

Reply:
I added a comment that they can remove it and saying why it's here.